docs/sources/panels-visualizations/query-transform-data/expression-queries/index.md

```diff
@@ -219,3 +219,7 @@ For more information about expressions, refer to [About expressions]({{< relref
 
 1. Write the expression.
 1. Click **Apply**.
+
+## Special cases
+When data source returns no data points the expression engine produces special result `NoData`.  All operations with this type of result result in NoData. If for example, the request contains two queries to data source that are merged by math expression, NoData returned by at least one of data source queries will casue the result of the math expression and all subsequent chain of expressions to be NoData.
```

Comment:
> When data source returns no data points the expression engine produces special result `NoData`.

I think it might be more accurate to say "no data items", or perhaps "no series or numbers", as there could be a Series with no datapoints in it, and I don't think this gets classified as "No Data".

Ref: https://github.com/grafana/grafana-plugin-sdk-go/tree/main/data/contract_docs#no-data-and-empty